examples/hosts/app-integration/external-data/src/mock-external-data-service/externalDataSource.ts

```diff
@@ -27,6 +27,18 @@ const startingExternalData: TaskData = {
 	},
 };
 
+const taskList2: TaskData = {
+	17: {
+		name: "Alpha",
+		priority: 1,
+	},
+};
+
+const startingExternalData: TaskListData = {
+	"task-list-1": taskList1,
```

Comment:
One thing to be careful about - is "task-list-1" the way the external data service refers to the data?  Or is it how the Fluid TaskList will refer to it?  If the former then this is fine, but if the latter then the external data source probably shouldn't know these IDs.

Below these same-looking IDs are used in the Fluid bits, so prob just make sure you're being intentional about where they are legitimate identifiers.

Reply:
> is "task-list-1" the way the external data service refers to the data? \

Yes, the intent is to make the ExternalData Service be the owner of the task list UUID and only reference it within Fluid, so they are all taking the same TaskListID.

Reply:
Gotcha, that makes sense.